qdyn/output.f90

```diff
@@ -1272,7 +1502,9 @@ subroutine pb_global(pb)
     ! Skip fault potency (rate)
     ! TODO: replace this hardcoded stuff with something
     ! more elegant...
-    if (i == 10 .or. i == 11) cycle
+    ! if (i == 10 .or. i == 11) cycle
+    ! CRP: skip also vmax_fault?
```

Comment:
I don't remember why we decided to skip fault pot and pot rate. Is this a temporary solution? Why does `vmax_fault` need to be skipped?

Reply:
Is it not because where the gathering of the nodes to the master processor in `pb_global` (where this line is) is called before calculating the potency per fault in `calc_potency(pb)` and getting the vmax per fault in `get_ivmax_fault(pb)`?

Reply:
True. Could you remove the commented lines?

Reply:
yes, removed